FogCreek/Glitch-Community
# Mammoth iguana: fix bug in deleting projects

## Links
* Remix link: https://mammoth-iguana.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/484

## Changes:
* don't overwrite _deletedProjects when updating the user

## How To Test:
* visit your profile page on glitch.com
* delete a project **without expanding the deleted projects section**
* the page should crash

* log into https://mammoth-iguana.glitch.me and visit your profile page
* delete a project
* the project should be removed from your recent projects
* open the deleted projects section
* the project should be in your deleted projects
